packages/assistify-ai/server/methods/SmartiWidgetBackend.js

```diff
@@ -185,3 +186,51 @@ Meteor.methods({
 		return script;
 	}
 });
+
+Meteor.methods({
+	triggerFullResync() {
+		const requests = RocketChat.models.Rooms.model.find({t:'r', closer: {$exists: false}}).fetch();
```

Comment:
why restrict to `closer: {$exists: false}}`
We're also interested in getting closed conversations synced